core/signeddata_test.go

```diff
@@ -67,3 +68,20 @@ func TestSetVersionedValidatorRegistrationSig(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEqual(t, clone.Signature(), registration.Signature())
 }
+
+func TestSetBeaconCommitteeSubscription(t *testing.T) {
```

Comment:
can this be refactored into a table test?

Reply:
done